src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  let sum = days * 40;
+
+  if (sum < 120) {
+    return sum;
+  }
+
+  if (sum >= 280) {
+    sum -= 50;
+
+    return sum;
+  }
+
+  if (sum >= 120) {
+    sum -= 20;
+
+    return sum;
+  }
```

Comment:
The condition `sum >= 120` is using a hardcoded value. It's better to use named constants to represent the threshold for the short-term rental discount, and another constant for the discount amount. Additionally, this condition is not necessary since if the code reaches this point, it implies that the previous condition was not met, and therefore the sum is already known to be less than the threshold for the long-term discount.